action.yml

```
@@ -29,7 +34,9 @@ runs:
   - id: generate-image-name
     shell: bash
     run: |
-      if [[ ${{ inputs.context }} == "." ]] || \
+      if [[ ${{ inputs.image_name }} != '' ]]; then
+          IMAGE_NAME="${{inputs.image_name}}"  
```

```suggestion
          IMAGE_NAME="${{ inputs.image_name }}"
```

Just to match the style of other references to input vars and remove trailing whitespace